index.html

```diff
@@ -181,22 +190,24 @@ <h1 class="about">About Myself</h1>
     <div>
         <input type="button" name="boton1" value="Get my resume" class="input-6">
     </div>
-</section>
+</div>
 
-<!--Seventh part-->
 
-<section class="conatiner-7">
-    <div class="languajes">
-    <h2>Languajes</h2>
-    <button class="down">
-    <i class="fa-solid fa-angle-down"></i>
-    </button>
-    </div>
+<!--Seventh part
+
+    <section class="container-7">-->
```

Comment:
- [ ] This section (Languages, Skills, Frameworks) should move a little bit to the right, please consider styling your page as Figma design so it looks more professional
- [ ] You have a typo here `<h2 class="lan">Languajes</h2>` `Languajes` should be `Languages`, kindly fix this so it's more professional

|Your design|Figma design|
|-|-|
|![Screenshot_20220611_093247](https://user-images.githubusercontent.com/25227022/173180706-d46f5542-010f-4c3a-b743-f87fd7082638.png)|![Screenshot_20220611_093310](https://user-images.githubusercontent.com/25227022/173180707-7eb23bce-c7fc-4ed6-9873-57323a1755a0.png)|